process/modules/converb_to_conj.py

```diff
@@ -0,0 +1,180 @@
+from process.module import ParaphraseModule
+from process.preprocessing_utils import PreprocessingUtils
+from stanza.utils.conll import CoNLL
+
+import stanza
+import pymorphy2 
+morph = pymorphy2.MorphAnalyzer()
+import pyconll
+import udon2
+
+class ConverbToConjuctionModule(ParaphraseModule):
+    def __init__(self, name="capitalize_nouns") -> None:
+        super().__init__(name=name)
+    
+    def load(self, preproc_utils: PreprocessingUtils) -> None:
+        # load any tools as `preproc_utils` attributes
+        preproc_utils.stanza_model = stanza.Pipeline('ru', processors='tokenize,pos,lemma,depparse')
```

Comment:
```suggestion
        if getattr(preproc_utils, 'stanza_model', None) is None:
            preproc_utils.stanza_model = stanza.Pipeline('ru', processors='tokenize,pos,lemma,depparse')
```

Лучше проверять, что станза еще не загружена, чтобы загрузка не вызывалась повторно в разных модулях.